htdocs/contact/card.php

```diff
@@ -474,9 +475,6 @@
         // Si edition contact deja existant
         $object = new Contact($db);
         $res=$object->fetch($id, $user);
-        if ($res < 0) { dol_print_error($db, $object->error); exit; }
```

Comment:
Why removing error reporting ?

Reply:
True, initially I want to remove fetch_optional that is now into contact class fetch method. I've gone too far with this removing. Just send a commit with fix.